Kudu.Services/Docker/DockerController.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Kudu.Core.Helpers;
+using Kudu.Core.Tracing;
+using System;
+using System.IO;
+using System.Net;
+using System.Net.Http;
+using System.Web.Http;
+
+namespace Kudu.Services.Docker
+{
+    public class DockerController : ApiController
+    {
+        const string TimestampDirectory = "/home/config";
```

Comment:
Does the functionality require that the timestamp file is in the mounted site directory? If so we'll want to keep a close eye on this, as a feature that may not be too far off is allowing the site drive to be mounted somewhere besides /home.